### PR TITLE
Show active keybindings in Help

### LINKED ISSUE #572: Show actual keybindings in help page
Currently, unless I'm mistaken, the help page shows hardcoded key bindings.

It would be nice to show the actual ones, loaded from the config file.

### PR DESCRIPTION
**Pull request by bi1yeu**
Closes #572 

Makes the Help page display the active keybindings from config rather than the hard-coded defaults.
 
Functionally tested by putting some keybindings in `~/.config/spotify-tui/config.yml` and observing they were displayed on the Help page.

I tried to match the current Help page styling for key strings: `<Enter>`, `<Space>`, `<Ctrl+x>`, etc.

active keybindings

P.S. This is my first time writing Rust, so please let me know if anything isn't idiomatic (or is completely wrong)!

### CODE REVIEW

File: src/event/key.rs [hunk 1]
```diff
@@ -94,6 +95,31 @@ impl Key {
...
+      | Key::Ins
+      | Key::Delete
+      | Key::Home
+      | Key::End
+      | Key::PageUp
+      | Key::PageDown => write!(f, "<{:?}>", self),
+      _ => write!(f, "{:?}", self),
+    }
+  }
+}
```
Comment (Rigellute) [inline src/event/key.rs:121]: Very nice 👌

File: src/ui/help.rs [hunk 2]
```diff
@@ -1,111 +1,261 @@
...
     ],
-    vec!["Jump to start of playlist", "<Ctrl+a>", "Pagination"],
-    vec!["Jump to end of playlist", "<Ctrl+e>", "Pagination"],
-    vec!["Jump to currently playing album", "a", "General"],
     vec![
-      "Jump to currently playing artist's album list",
-      "A",
-      "General",
+      String::from("Scroll up to previous result page"),
+      String::from("<Ctrl+u>"),
```
Comment (Rigellute) [inline src/ui/help.rs:12]: Scrolling up/down and jumping start/end are now added to the config - could display those values too?

This PR just got merged adding these to the config https://github.com/Rigellute/spotify-tui/pull/566

File: src/ui/help.rs [hunk 3]
```diff
@@ -1,111 +1,261 @@
...
+    ],
+    vec![
+      String::from("Save (like) album to library"),
+      String::from("w"),
+      String::from("Search result"),
+    ],
+    vec![
+      String::from("Play random song in playlist"),
+      String::from("S"),
+      String::from("Selected Playlist"),
```
Comment (Rigellute) [inline src/ui/help.rs:258]: 👍

**Review by Rigellute — APPROVED** [review]
Amazing work @bi1yeu!! 

I've been wanting this feature myself. A very nice addition 👏.

If you wouldn't mind adding the new recently merged keybindings, I'll merge.

File: src/event/key.rs (hunk 1 above)
Comment (bi1yeu) [reply src/event/key.rs:121]: One note about this: there aren't any default keybindings currently bound to function keys, so I wasn't sure whether those should be formatted as `F12` or `<F12>`. If you prefer the latter then you can do away with the `Key::Enter...` case above and make the default case `_ => write!(f, "<{:?}>", self),`.

**Comment by bi1yeu** [comment]
> If you wouldn't mind adding the new recently merged keybindings, I'll merge.

I merged in upstream changes and added the missing keybindings. Also left a comment re: displaying function keys. This should be ready to go now -- thanks for reviewing!

File: src/event/key.rs (hunk 1 above)
Comment (Rigellute) [reply src/event/key.rs:121]: Good to know, thank you.

**Review by Rigellute — APPROVED** [review]
Brilliant work @bi1yeu, thanks again.

**Comment by Rigellute** [comment]
@all-contributors please add @bi1yeu for code, doc

**Pull request merged**